buildscripts/ci/windows/sign_test.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
+setlocal ENABLEDELAYEDEXPANSION
```

Comment:
What does this do?

Reply:
in fact it's copy-paste and I didn't find out